Issue again with `xo` handling Typescript definitions

```bash
$ npm t

> load-json-file@5.0.0 test /Users/denis/Github/load-json-file
> xo && ava


  index.d.ts:1:1
  ✖  1:1  Parsing error: The keyword interface is reserved

  1 error
npm ERR! Test failed.  See above for more details.
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -11,7 +11,7 @@ interface Options {
 }
 
 /**
- * Read and parse a JSON file
+ * Read and parse a JSON file.
  *
  * Strips UTF-8 BOM, uses graceful-fs, and throws more helpful JSON errors.
  *
@@ -25,7 +25,7 @@ interface Options {
 export function sync(filepath: string, options?: Options): void;
 
 /**
- * Read and parse a JSON file
+ * Read and parse a JSON file.
  *
  * Strips UTF-8 BOM, uses graceful-fs, and throws more helpful JSON errors.
  *

--- a/index.d.ts
+++ b/index.d.ts
@@ -15,15 +15,9 @@ interface Options {
  *
  * Strips UTF-8 BOM, uses graceful-fs, and throws more helpful JSON errors.
  *
- * @param {string} filepath Filepath
- * @param {any} data Data
- * @param {object} [options] Optional parameters
- * @param {function} [options.beforeParse] Applies a function to the JSON string before parsing.
- * @param {function} [options.reviver] Prescribes how the value originally produced by parsing is transformed, before being returned.
- * See the [JSON.parse docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse#Using_the_reviver_parameter) for more.
  * @returns {void}
  * @example
- * const loadJsonFile = require('load-json-file');
+ * import * as loadJsonFile from 'load-json-file';
  *
  * loadJsonFile.sync('foo.json')
  * console.log(json);
@@ -35,19 +29,13 @@ export function sync(filepath: string, options?: Options): void;
  *
  * Strips UTF-8 BOM, uses graceful-fs, and throws more helpful JSON errors.
  *
- * @param {string} filepath Filepath
- * @param {any} data Data
- * @param {object} [options] Optional parameters
- * @param {function} [options.beforeParse] Applies a function to the JSON string before parsing.
- * @param {function} [options.reviver] Prescribes how the value originally produced by parsing is transformed, before being returned.
- * See the [JSON.parse docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse#Using_the_reviver_parameter) for more.
  * @returns {Promise<void>}
  * @example
- * const loadJsonFile = require('load-json-file');
+ * import loadJsonFile 'load-json-file';
  *
- * loadJsonFile('foo.json').then(json => {
- * 	console.log(json);
- * 	//=> {foo: true}
- * });
+ * (async () => {
+ *     await loadJsonFile('foo.json');
+ *     console.log('done');
+ * })();
  */
 export default function loadJsonFile(filepath: string, options?: Options): Promise<void>;

--- a/index.js
+++ b/index.js
@@ -15,6 +15,8 @@ const parse = (data, filePath, options = {}) => {
 	return parseJson(data, options.reviver, path.relative(process.cwd(), filePath));
 };
 
-module.exports = (filePath, options) => pify(fs.readFile)(filePath, 'utf8').then(data => parse(data, filePath, options));
-module.exports.default = (filePath, options) => pify(fs.readFile)(filePath, 'utf8').then(data => parse(data, filePath, options));
+const loadJsonFile = (filePath, options) => pify(fs.readFile)(filePath, 'utf8').then(data => parse(data, filePath, options));
+
+module.export = loadJsonFile;
+module.exports.default = loadJsonFile;
 module.exports.sync = (filePath, options) => parse(fs.readFileSync(filePath, 'utf8'), filePath, options);

--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
 		"node": ">=6"
 	},
 	"scripts": {
-		"test": "xo && ava"
+		"test": "xo --ignore index.d.ts && ava"
 	},
 	"files": [
 		"index.js",

--- a/package.json
+++ b/package.json
@@ -19,7 +19,6 @@
 		"index.js",
 		"index.d.ts"
 	],
-	"types": "index.d.ts",
 	"keywords": [
 		"read",
 		"json",

--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
 		"node": ">=6"
 	},
 	"scripts": {
-		"test": "xo --ignore index.d.ts && ava"
+		"test": "xo && ava"
 	},
 	"files": [
 		"index.js",
@@ -37,5 +37,10 @@
 	"devDependencies": {
 		"ava": "*",
 		"xo": "*"
+	},
+	"xo": {
+		"ignores": [
+			"index.d.ts"
+		]
 	}
 }

